x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.ts

```diff
@@ -51,7 +51,6 @@ interface RelevanceTuningActions {
   setResultsLoading(resultsLoading: boolean): boolean;
   clearSearchResults(): void;
   resetSearchSettingsState(): void;
-  dismissSchemaConflictCallout(): void;
```

Comment:
I removed this entire concept, of dismissing this prompt because it just added extra complexity for very little value. No other errors are dismissable, so why this one? The dismissal isn't even persisted, it comes back as soon as you navigate away and back.